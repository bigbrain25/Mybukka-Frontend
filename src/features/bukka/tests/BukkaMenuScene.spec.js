import React from 'react';
import BukkaMenu from '../components';

describe('BukkaMenu scene', () => {
  const wrapper = shallow(<BukkaMenu />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
