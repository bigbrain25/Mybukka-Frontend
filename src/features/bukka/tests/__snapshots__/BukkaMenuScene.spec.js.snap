// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BukkaMenu scene renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BukkaMenuScene
    push={[Function]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <AddToCart
          imageUrl=""
        />,
        <PrimaryNavbar
          push={[Function]}
        />,
        <BukkaImage
          imageUrl="https://res.cloudinary.com/dn93xk5ni/image/upload/v1549932720/bake-baked-basil-236798_vvo5pq.jpg"
        />,
        <BukkaDetailsSection
          bukkaName="Chipottle Fresh Bukka"
          description=""
        />,
        <BukkaMeals
          mealsData={
            Array [
              Object {
                "category": "Breakfast",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "sample meal",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Breakfast",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "sample meal",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Breakfast",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "sample meal",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Breakfast",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "sample meal",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Dinner",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "Sagalita Burritos",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Dinner",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "Sampalina Gurrima",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Cuisines",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "Cuisinola",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Cuisines",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "Cuisinola",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Cuisines",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "Cuisinola",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Cuisines",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "Cuisinola",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Cuisines",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "Cuisinola",
                "price": 30,
                "title": "Amala",
              },
            ]
          }
        />,
        <Footer />,
      ],
      "className": "bukka-menu",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "imageUrl": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "push": [Function],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549932720/bake-baked-basil-236798_vvo5pq.jpg",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "bukkaName": "Chipottle Fresh Bukka",
          "description": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "mealsData": Array [
            Object {
              "category": "Breakfast",
              "description": "A very great plate of Amala that will make you beg for more",
              "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
              "name": "sample meal",
              "price": 30,
              "title": "Amala",
            },
            Object {
              "category": "Breakfast",
              "description": "A very great plate of Amala that will make you beg for more",
              "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
              "name": "sample meal",
              "price": 30,
              "title": "Amala",
            },
            Object {
              "category": "Breakfast",
              "description": "A very great plate of Amala that will make you beg for more",
              "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
              "name": "sample meal",
              "price": 30,
              "title": "Amala",
            },
            Object {
              "category": "Breakfast",
              "description": "A very great plate of Amala that will make you beg for more",
              "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
              "name": "sample meal",
              "price": 30,
              "title": "Amala",
            },
            Object {
              "category": "Dinner",
              "description": "A very great plate of Amala that will make you beg for more",
              "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
              "name": "Sagalita Burritos",
              "price": 30,
              "title": "Amala",
            },
            Object {
              "category": "Dinner",
              "description": "A very great plate of Amala that will make you beg for more",
              "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
              "name": "Sampalina Gurrima",
              "price": 30,
              "title": "Amala",
            },
            Object {
              "category": "Cuisines",
              "description": "A very great plate of Amala that will make you beg for more",
              "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
              "name": "Cuisinola",
              "price": 30,
              "title": "Amala",
            },
            Object {
              "category": "Cuisines",
              "description": "A very great plate of Amala that will make you beg for more",
              "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
              "name": "Cuisinola",
              "price": 30,
              "title": "Amala",
            },
            Object {
              "category": "Cuisines",
              "description": "A very great plate of Amala that will make you beg for more",
              "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
              "name": "Cuisinola",
              "price": 30,
              "title": "Amala",
            },
            Object {
              "category": "Cuisines",
              "description": "A very great plate of Amala that will make you beg for more",
              "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
              "name": "Cuisinola",
              "price": 30,
              "title": "Amala",
            },
            Object {
              "category": "Cuisines",
              "description": "A very great plate of Amala that will make you beg for more",
              "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
              "name": "Cuisinola",
              "price": 30,
              "title": "Amala",
            },
          ],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <AddToCart
            imageUrl=""
          />,
          <PrimaryNavbar
            push={[Function]}
          />,
          <BukkaImage
            imageUrl="https://res.cloudinary.com/dn93xk5ni/image/upload/v1549932720/bake-baked-basil-236798_vvo5pq.jpg"
          />,
          <BukkaDetailsSection
            bukkaName="Chipottle Fresh Bukka"
            description=""
          />,
          <BukkaMeals
            mealsData={
              Array [
                Object {
                  "category": "Breakfast",
                  "description": "A very great plate of Amala that will make you beg for more",
                  "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                  "name": "sample meal",
                  "price": 30,
                  "title": "Amala",
                },
                Object {
                  "category": "Breakfast",
                  "description": "A very great plate of Amala that will make you beg for more",
                  "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                  "name": "sample meal",
                  "price": 30,
                  "title": "Amala",
                },
                Object {
                  "category": "Breakfast",
                  "description": "A very great plate of Amala that will make you beg for more",
                  "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                  "name": "sample meal",
                  "price": 30,
                  "title": "Amala",
                },
                Object {
                  "category": "Breakfast",
                  "description": "A very great plate of Amala that will make you beg for more",
                  "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                  "name": "sample meal",
                  "price": 30,
                  "title": "Amala",
                },
                Object {
                  "category": "Dinner",
                  "description": "A very great plate of Amala that will make you beg for more",
                  "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                  "name": "Sagalita Burritos",
                  "price": 30,
                  "title": "Amala",
                },
                Object {
                  "category": "Dinner",
                  "description": "A very great plate of Amala that will make you beg for more",
                  "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                  "name": "Sampalina Gurrima",
                  "price": 30,
                  "title": "Amala",
                },
                Object {
                  "category": "Cuisines",
                  "description": "A very great plate of Amala that will make you beg for more",
                  "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                  "name": "Cuisinola",
                  "price": 30,
                  "title": "Amala",
                },
                Object {
                  "category": "Cuisines",
                  "description": "A very great plate of Amala that will make you beg for more",
                  "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                  "name": "Cuisinola",
                  "price": 30,
                  "title": "Amala",
                },
                Object {
                  "category": "Cuisines",
                  "description": "A very great plate of Amala that will make you beg for more",
                  "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                  "name": "Cuisinola",
                  "price": 30,
                  "title": "Amala",
                },
                Object {
                  "category": "Cuisines",
                  "description": "A very great plate of Amala that will make you beg for more",
                  "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                  "name": "Cuisinola",
                  "price": 30,
                  "title": "Amala",
                },
                Object {
                  "category": "Cuisines",
                  "description": "A very great plate of Amala that will make you beg for more",
                  "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                  "name": "Cuisinola",
                  "price": 30,
                  "title": "Amala",
                },
              ]
            }
          />,
          <Footer />,
        ],
        "className": "bukka-menu",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "imageUrl": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "push": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549932720/bake-baked-basil-236798_vvo5pq.jpg",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "bukkaName": "Chipottle Fresh Bukka",
            "description": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "mealsData": Array [
              Object {
                "category": "Breakfast",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "sample meal",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Breakfast",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "sample meal",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Breakfast",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "sample meal",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Breakfast",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "sample meal",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Dinner",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "Sagalita Burritos",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Dinner",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "Sampalina Gurrima",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Cuisines",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "Cuisinola",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Cuisines",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "Cuisinola",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Cuisines",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "Cuisinola",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Cuisines",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "Cuisinola",
                "price": 30,
                "title": "Amala",
              },
              Object {
                "category": "Cuisines",
                "description": "A very great plate of Amala that will make you beg for more",
                "imageUrl": "https://res.cloudinary.com/dn93xk5ni/image/upload/v1549933392/burrito-chicken-close-up-461198_g9llka.jpg",
                "name": "Cuisinola",
                "price": 30,
                "title": "Amala",
              },
            ],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
